packages/elixir/build.sh

```diff
@@ -3,8 +3,10 @@ TERMUX_PKG_DESCRIPTION="Elixir is a dynamic, functional language designed for bu
 TERMUX_PKG_LICENSE="Apache-2.0"
 TERMUX_PKG_MAINTAINER="@termux"
 TERMUX_PKG_VERSION="1.17.2"
-TERMUX_PKG_SRCURL=https://github.com/elixir-lang/elixir/releases/download/v${TERMUX_PKG_VERSION}/elixir-otp-25.zip
-TERMUX_PKG_SHA256=6ca8215fd8b1fa4472fa62c8258d3c2880145070bf04848f51861c7d1d219c7e
+TERMUX_PKG_REVISION=1
+_ERLANG_MAJOR_VERSION=$(. $TERMUX_SCRIPTDIR/packages/erlang/build.sh; echo ${TERMUX_PKG_VERSION%%.*})
```

Comment:
Quote yo vars.
```suggestion
_ERLANG_MAJOR_VERSION=$(. "$TERMUX_SCRIPTDIR"/packages/erlang/build.sh; echo "${TERMUX_PKG_VERSION%%.*}")
```
But that'll do it.

Reply:
Done.

Reply:
The command substitution itself should also be quoted.
Which, I recognize is overkill.
But we may as well do our quoting properly.

Reply:
> Quote yo vars.
> 
> But that'll do it.